content/docs/migration_1.0/Concepts.md

```diff
@@ -0,0 +1,126 @@
+You can take a look at examples of usage in any language’s examples/z_pull.x
+
+## Timestamps
+
+We now tie generating a timestamp to a Zenoh session, with the timestamp inheriting the `ZenohID` of the session.
+
+This will affect user-created plugins and applications that need to generate timestamps in their Storage and sending of data.  
+⚠️ Note: Timestamps are important for Storage Alignment and Replication. Data stored in Data bases must include a Timestamp to be properly aligned across Data Stores by Zenoh. 
+The `timestamping` configuration option must also be enabled for this.
+
+## Storages
+The storage-manager will fail to launch if the timestamping configuration option is disabled.  
+From Zenoh 1.0.0 user-applications can load plugins.  
+A, somehow, implicit assumption that dictated the behaviour of storages is that the Zenoh node loading them **has to add a timestamp to any received publication that did not have one**. This functionality is controlled by the `timestamping` configuration option.  
+Until Zenoh 1.0.0 this assumption held true as only a router could load storage and the default configuration for a router enables `timestamping`. However, in Zenoh 1.0.0 nodes configured in `client` & `peer` mode can load storage and *their default configuration disables `timestamping`*.
+
+## Config Changes
+
+### Plugin Loading
+
+We added the ability for user-applications to load compiled plugins written in Rust, regardless of  language ! 
+
+Usage of this feature is achieved by simply enabling the `plugins_loading` section in config file with the members `enabled` set to true, and specifying the `search_dirs` for the plugins. 
+
+If no search directories were specified, then the default search directories are 
+`".:~/.zenoh/lib:/opt/homebrew/lib:/usr/local/lib:/usr/lib”` 
+
+```jsx
+ plugins_loading: {
+    // Enable plugins loading.
+    enabled: false,
+    /// Directories where plugins configured by name should be looked for. Plugins configured by __path__ are not subject to lookup.
+    /// If `enabled: true` and `search_dirs` is not specified then `search_dirs` falls back to the default value: ".:~/.zenoh/lib:/opt/homebrew/lib:/usr/local/lib:/usr/lib"
+    search_dirs: [],
+}
+// ... Rest of Config 
+```
+
+⚠️ Note : When loading a plugin, the Plugin must have been built with the same version of the Rust compiler as the bindings loading it. 
+This means that if the language bindings are using `rustc` version `1.75`, the plugin must be built with the same version.
+
+### Scouting
+ 
+We have implemented a small change in configuration syntax concerning the `scouting` section of config files.   
+Both `gossip` and `multicast`’s `autoconnect` section's have changed to accept lists of either 
+`"peer"`, `"client"` or `"router"`
+
+```jsx
+// Zenoh 0.11.0
+scouting: {
+  multicast: {
+    autoconnect: { router: "", peer: "router|peer" },
+  },
+  gossip: {
+    autoconnect: { router: "", peer: "router|peer" },
```

Comment:
Update the config to `1.0.0` format: https://github.com/eclipse-zenoh/zenoh/blob/e38fc16b9988c5c6add6d2e19667b6badc86993d/DEFAULT_CONFIG.json5#L110